src/main/java/com/deque/axe/android/rules/hierarchy/base/TouchSize.java

```diff
@@ -10,6 +10,7 @@
 public abstract class TouchSize extends ActiveView {
 
   private final int expectedSize;
+  private AxeDevice axeDevice;
```

Comment:
There is no assignment being done on the axeDevice, it should be fine to keep it as final even as a member variable.

Reply:
`axeDevice` is being initialized in `setup` method